onnxruntime/core/optimizer/qdq_transformer/qdq_util.cc

```diff
@@ -54,7 +54,7 @@ bool IsQDQPairSupported(
   Initializer dq_scale(*dq_scale_tensor_proto, model_path);
 
   return q_zp.data_type() == dq_zp.data_type() &&
-         *q_zp.data<int8_t>() == *dq_zp.data<int8_t>() &&
+         *q_zp.raw_data() == *dq_zp.raw_data() &&
```

Comment:
>raw_data [](http://example.com/codeflow?start=15&length=8)

does `Initializer::data<int8_t>()` also check that the dtype is int8?

Reply:
Will check

Reply:
The data is sometimes int8_t but mostly uint8_t, however, the code disregards it.